openshift/capabilities-tool-job.yaml

```diff
@@ -7,8 +7,8 @@ spec:
     spec:
       containers:
       - name: capabilities-tool
-        image: quay.io/opdev/capabilities-tool:v0.2.7
-        args: ["index","capabilities","--container-engine","podman","--output-path","/opt/capabilities-tool","--bundle-image","$(bundle_image)","--bucket-name","$(bucket_name)","--bundle-name","$(bundle_name)"]
+        image: quay.io/opdev/capabilities-tool:v1.0.1-test
```

Comment:
Why did the image name changed here to a lower number with -test? Didn't get this one.